static/app/views/replays/detail/detailLayout.tsx

```diff
@@ -155,7 +186,13 @@ const BigNameUserBadge = styled(UserBadge)`
 
 // TODO(replay); This could make a lot of sense to put inside HeaderActions by default
 const FeedbackButtonWrapper = styled(Layout.HeaderActions)`
-  align-items: end;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: flex-start;
+`;
+
+const ShareButton = styled(Button)`
+  margin-right: 8px;
```

Comment:
Instead of adding a margin to the button, let's change `FeedbackButtonWrapper` to a grid and use a `gap`.

Also instead of hard coding the pixel, use `space()`.